### DISCUSSION TITLE
Is a friendly match with a different branch disabled?

### DISCUSSION THREAD

**Discussion by YuriCat** [cyberbotics/wrestling#107] [Q&A] [answered]
Dear organizer,

Thank you for this wonderful competition!

I requested a friendly match from a branch other than **main**, but a regular match for ranking with **main** branch took place.

Is it not possible to use a friendly match from another branch?

**Comment by omichel** [comment, marked as answer]
No, this is unfortunately not supported at the moment.

**Reply by YuriCat** [reply]
I see, thank you for your clear answer.
